.github/CODEOWNERS

```diff
@@ -106,75 +106,121 @@
 /pkg/sql/schema*.go          @cockroachdb/sql-schema
 /pkg/sql/zone*.go            @cockroachdb/sql-schema
 
-/pkg/cli/                    @cockroachdb/cli-prs
-# last-rule-wins so bulk i/o takes userfile.go even though cli-prs takes pkg/cli
-/pkg/cli/userfile.go         @cockroachdb/disaster-recovery
+#!/pkg/cli/                  @cockroachdb/unowned
 /pkg/cli/auth.go             @cockroachdb/prodsec @cockroachdb/cli-prs
-/pkg/cli/cert*.go            @cockroachdb/cli-prs        @cockroachdb/prodsec
-/pkg/cli/demo*.go            @cockroachdb/sql-sessions @cockroachdb/server-prs @cockroachdb/cli-prs
-/pkg/cli/democluster         @cockroachdb/sql-sessions @cockroachdb/server-prs @cockroachdb/cli-prs
-/pkg/cli/debug*.go           @cockroachdb/kv-prs         @cockroachdb/cli-prs
-/pkg/cli/debug_job_trace*.go @cockroachdb/jobs-prs @cockroachdb/disaster-recovery
-/pkg/cli/doctor*.go          @cockroachdb/sql-schema     @cockroachdb/cli-prs
-/pkg/cli/import_test.go      @cockroachdb/sql-sessions @cockroachdb/cli-prs
-/pkg/cli/sql*.go             @cockroachdb/sql-sessions @cockroachdb/cli-prs
-/pkg/cli/clisqlshell/        @cockroachdb/sql-sessions @cockroachdb/cli-prs
-/pkg/cli/clisqlclient/       @cockroachdb/sql-sessions @cockroachdb/cli-prs
+/pkg/cli/cert*.go            @cockroachdb/prodsec @cockroachdb/cli-prs
+/pkg/cli/cli.go              @cockroachdb/cli-prs
```

Comment:
Nit: I'm wondering why the assignment to `cli-prs` is good here but not as the primary assignment in other places. A few words about that as a comment or in the PR description might be helpful.

Reply:
The rationale is that these are individual files. The assignment to the _directory_ makes the team responsible for any _new_ files added in the future, as well as files that already exist but not assigned explicitly in CODEOWNERS. That's what we want to avoid. I'll add a comment.